### PR TITLE
[14.0][ADD] stock_prelocation_putaway

### PR DESCRIPTION
**Pull request by hparfr**
Add module stock_location_putaway

It's a simple way to mimic putaway rules in reception.

### CODE REVIEW

File: stock_prelocation_putaway/README.rst [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/285/14.0
+    :alt: Try me on Runbot
+
+|badge1| |badge2| |badge3| |badge4| |badge5| 
+
+This module allows to use pre-sort locations.
+
+For instance, in a wherehouse with alleys named A,B,C and D:
```
Comment (bealdav) [inline stock_prelocation_putaway/README.rst:30]: `wherehouse` c pas rillieux de janeiro chez toi ?

Comment (hparfr) [reply stock_prelocation_putaway/README.rst:30]: Fixed thanks ;)

**Comment by hparfr** [comment]
@bealdav the depencies are now merged. You can update your review thanks ;-)

**Review by bealdav — APPROVED** [review]

File: stock_prelocation_putaway/README.rst (hunk 1 above)
Comment (sebalix) [reply stock_prelocation_putaway/README.rst:30]: Still not fixed it seems, not pushed? :)

Comment (hparfr) [reply stock_prelocation_putaway/README.rst:30]: It's fixed in DESCRIPTION.rst line:3. The readme has not bee re-generated.

**Review by bguillot — APPROVED** [review]

**Comment by sebalix** [comment]
@jbaudoux is it something useful? Or already addressed by `stock_dynamic_routing` somehow (I'm maybe wrong)?

**Comment by jbaudoux** [comment]
> @jbaudoux is it something useful? Or already addressed by `stock_dynamic_routing` somehow (I'm maybe wrong)?

@sebalix This is very likely covered by the stock_storage_type module. By assigning a storage type on a product, you can manage putaway rules on any location.

So instead of defining a product putaway rule "WH/STOCK -> WH/STOCK/A"
you assign a storage type "Zone A", then you can define rules for that storage type:
- STOCK > STOCK/A
- INPUT > INPUT/AB

You can also do far more advanced setup with stock_storage_type.

**Comment by hparfr** [comment]
Indeed, you probably don't need it for your advanced setup. Thanks for the explanation @jbaudoux

**Pull request closed without merging**